tests/test_key_revocation.py

```diff
@@ -385,9 +388,6 @@ def test_root_key_revocation(self):
       for mirror_exception in exception.mirror_errors.values():
         self.assertTrue(isinstance(mirror_exception, tuf.BadSignatureError))
 
-    # Load the previous Root signing key so that the the client can update
-    # successfully.
-    repository.root.load_signing_key(self.role_keys['root']['private'])
```

Comment:
I'll think about how to improve this test and update. Without changing the location, the updater can't update to the next version at all - signing a version without a previous root key breaks the chain for all subsequent versions.